content/src/main/content/jcr_root/apps/acs-commons/authoring/sitewide-component-props/sitewide-component-props.js

```diff
@@ -0,0 +1,51 @@
+(function ($, ns, channel, window, undefined) {
+
+    var actionDef = {
+        icon: 'coral-Icon--globe',
+        text: Granite.I18n.get('Configure Site-wide Component Properties'),
+        handler: function (editable, param, target) { // will be called on click
+            var originalDialogSrc = editable.config.dialogSrc;
+            var originalDialog = editable.config.dialog;
+            
+            var dialogSrcArray = editable.config.dialogSrc.split(".html");
+            var langRootRegexp = /^(\/content\/([^/]+\/)+([a-zA-Z]{2}([_-][a-zA-Z]{2})?))(|\/.*)$/g;
+            var match = langRootRegexp.exec(dialogSrcArray[1]);
+            
+            var siteWideDialogSrc = dialogSrcArray[0].replace("_cq_dialog", "_cq_dialogsitewide") +
+                                    ".html" +
+                                    match[1] +
+                                    "/jcr:content/sitewideprops/" +
+                                    editable.type;
+            
+            editable.config.dialogSrc = siteWideDialogSrc;
+            editable.config.dialog = editable.config.dialog.replace("cq:dialog", "cq:dialogsitewide");
```

Comment:
can we use acs-commons:dialogsitewide instead of cq:dialogsitewide? In theory, only AEM should define node or property names using the 'cq' namespace.